.travis.yml

```diff
@@ -8,7 +8,15 @@ services:
   - postgresql
 install:
   - "pip install -e ."
-  - "pip install -r requirements-dev.txt
+  - "pip install -r requirements-dev.txt --use-wheel"
```

Comment:
You shouldn't need this or the `"` quotes. I guess you are just making this like the others, though.